packages/runtime/src/next/client.ts

```diff
@@ -39,12 +59,12 @@ export type MakeswiftPageDocument = {
     isIndexingBlocked: boolean | null
   }
 }
+type MakeswiftResources = ElementTreeResources & NonElementTreeResources
 
-export type MakeswiftPageSnapshot = {
-  document: MakeswiftPageDocument
-  apiOrigin: string
-  cacheData: CacheData
-  preview: boolean
+export type MakeswiftSnapshotV1 = {
+  resources: MakeswiftResources
+  elementTree: Element
+  runtimeVersion: string
```

Comment:
Using the `V1` suffix so we can support the old format in the meantime, get this merged, and break no ones app.